environment_osx.yml

```diff
@@ -28,7 +28,7 @@ dependencies:
   - fire
   - fsspec
   - gdal >=3.9.0
-  - importlib_metadata <8  # https://github.com/ESMValGroup/ESMValTool/issues/3699 only for Python 3.10/11 and esmpy<8.6
+  - importlib_metadata <8  # https://github.com/ESMValGroup/ESMValTool/issues/3699 only for Python 3.11 and esmpy<8.6
```

Comment:
I think this can be removed altogether since esmpy is blocked to <8.6 by NCL which OSX/pure Python dont have